lib/administrate/field/base.rb

```diff
@@ -27,6 +27,10 @@ def self.permitted_attribute(attr)
         attr
       end
 
+      def label
+        options[:label] || attribute.to_s.humanize.capitalize
```

Comment:
Sure, that would work as well - I don't think there's any reason to do it this way (other than that it is explicit).